test/services/sol_prices/api_client_test.rb

```diff
@@ -0,0 +1,4 @@
+module SolPrices
+  class ApiClientTest < ActiveSupport::TestCase
```

Comment:
send_request method should be tested

Reply:
I think it's enough tested in api_wrappers tests where is used.